web/react/components/get_link_modal.jsx

```diff
@@ -36,7 +36,7 @@ module.exports = React.createClass({
                           <h4 className="modal-title" id="myModalLabel">{this.state.title} Link</h4>
                         </div>
                         <div className="modal-body">
-                          <p>{"The link below is used for open " + strings.TeamPlural + " or if you allowed your " + strings.Team + " members to sign up using their " + strings.Company + " email addresses."}
+                          <p>{"Send teammates the link below to sign up for this team. Do not share this link publicly, since anyone with the link can join your team."}
```

Comment:
We need to use the variable ```strings.Team``` anywhere we use the word 'team' so that it replaces properly based on different configurations. So for here please use:

```<p>{"Send "+strings.Team+"mates the link below to sign up for this "+strings.Team+". Do not share this link publicly, since anyone with the link can join your "+strings.Team+"."}</p>```